index.html

```diff
@@ -57,6 +57,203 @@ <h1 id="primary-text">I'm Esteban. Glad to see you!</h1>
+        </div>
+        <div class="card work">
+          <img
+            class="card-img"
+            src="assets/works/Snapshoot Portfolio (2).png"
+            alt="project screenshot"
+          />
+          <h1 class="card-title">Multi-Post Stories</h1>
+          <div class="details-container">
+            <p class="client">CANOPY</p>
+            <img src="assets/works/Counter.png" alt="" />
+            <p class="role">Back End Dev</p>
+            <img src="assets/works/Counter.png" alt="" />
+            <p class="year">2015</p>
+          </div>
+          <p class="work-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="tags">
+            <li class="tag">HTML</li>
+            <li class="tag">CSS</li>
+            <li class="tag">JavaScript</li>
+          </ul>
+          <div class="action">
+            <button class="cta-btn">See project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about-container">
+        <div id="about">
+          <h1 class="primary-text">About Myself</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website. Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hesitate to contact me.
+          </p>
+          <p>LET'S CONNECT</p>
+          <ul class="socials">
+            <li id="twitter">
+              <a href="https://twitter.com/NaughTban"
+                ><img src="assets/headline/twitter.svg" alt="twitter icon"
+              /></a>
+            </li>
+            <li id="linkedin">
+              <a href="https://www.linkedin.com/in/esteban-palacios-5030a772/"
+                ><img src="assets/headline/linkedin.svg" alt="linkedin icon"
+              /></a>
+            </li>
+            <li id="medium">
+              <a href="/"
+                ><img src="assets/headline/medium.svg" alt="medium icon"
+              /></a>
+            </li>
+            <li id="github">
+              <a href="https://github.com/Estete9"
+                ><img src="assets/headline/github.svg" alt="github icon"
+              /></a>
+            </li>
+            <li id="angel-list">
+              <a href="/"
+                ><img src="assets/headline/angel list.svg" alt=""
+              /></a>
+            </li>
+          </ul>
+          <div id="resume-container">
+            <button class="cta-btn">Get my resume</button>
+          </div>
+        </div>
+        <ul id="skills-list">
+          <li class="skill">
+            <div>
+              <p>Languages</p>
+              <img
+                src="assets/about me/Icon - Chevron - Down - Larger.svg"
+                alt=""
+              />
+            </div>
+            <ul id="languages-list">
```

Comment:
- Hey, let's improve the design by adding some gaps to the container to the child element can be aligned correctly.
- Add a display of flex, flex-direction of column and a gap of 12px to fix the spacing issue as seen below:
*your design*
![image](https://github.com/Estete9/Fullstack-Portfolio/assets/59397159/8b86b783-6208-407d-b0cc-278528384b09)


*Figma design*
![image](https://github.com/Estete9/Fullstack-Portfolio/assets/59397159/0bd08244-3e94-4bf3-831b-1dcf960b58a0)